test/tests/Xray/XrayScanClient.spec.ts

```diff
@@ -55,7 +55,52 @@ describe('Scan graph tests', () => {
         await client
             .xray()
             .scan()
-            .graph({ component_id: 'engine' } as IGraphRequestModel, progress, () => undefined, 'ecosys');
+            .graph({ component_id: 'engine' } as IGraphRequestModel, progress, () => undefined, 'ecosys', []);
+        expect(scope.isDone()).toBeTruthy();
+        expect(progress.lastPercentage).toBe(100);
+    });
+
+    test('Watch test', async () => {
+        const uri: string = `/xray/api/v1/scan/graph`;
+        nock(PLATFORM_URL)
+            .post(uri + '?watch=watch-1')
+            .reply(200, { scan_id: '123' } as IGraphResponse);
+        const scope: nock.Scope = nock(PLATFORM_URL)
+            .get(uri + '/123?include_licenses=true&include_vulnerabilities=false')
+            .reply(200);
+        const client: JfrogClient = new JfrogClient({
+            platformUrl: PLATFORM_URL,
+            logger: TestUtils.createTestLogger(),
+        });
+        const progress: DummyProgress = new DummyProgress();
+        await client
+            .xray()
+            .scan()
+            .graph({ component_id: 'engine' } as IGraphRequestModel, progress, () => undefined, '', ['watch-1']);
+        expect(scope.isDone()).toBeTruthy();
+        expect(progress.lastPercentage).toBe(100);
+    });
+
+    test('Watches test', async () => {
+        const uri: string = `/xray/api/v1/scan/graph`;
```

Comment:
We can use `XrayScanClient.scanGraphEndpoint` static variable, instead of a raw text.